lib/sprite.dart

```diff
@@ -23,7 +24,13 @@ class Sprite {
 
   Vector2 get originalSize => Vector2(_imageWidth, _imageHeight);
 
-  Vector2 get size => Vector2(src.width, src.height);
+  Vector2 get size => Vector2(bounds.width, bounds.height);
+
+  Vector2 get position => bounds.topLeft.toVector2();
+
+  set position(Vector2 position) {
+    bounds = (position ?? Vector2.zero()).toPositionedRect(size);
```

Comment:
oh I see what you did. I don't think the use case existing sprite.position = null throwing an error is a major problem as it's clearly a mistake but I'm ok with handling this here rather than in the constructor.

Reply:
Now you can move the sprite position afterwards too buy just setting the position was my thought